distribution/scripts/setup/setup-start.sh

```diff
@@ -50,12 +50,20 @@ source $SCRIPTS_DIR/jmeter/install-jmeter.sh
 # Build components package
 $SCRIPTS_DIR/setup/build-components.sh
 
-ballerina_test_scenario="h1c-h1c-passthrough"
+#Run Jmeter test
+chmod +x $SCRIPTS_DIR/jmeter/start-jmeter-test.sh
+source $SCRIPTS_DIR/jmeter/start-jmeter-test.sh $JMETER_OPTIONS
 
-# Create Docker images and push to ECR
-$SCRIPTS_DIR/netty/make-netty-image.sh
-$SCRIPTS_DIR/ballerina/make-ballerina-image.sh -t $ballerina_test_scenario
-$SCRIPTS_DIR/jmeter/make-jmeter-image.sh
+#Create csv and md file
+cd $GITHUB_REPO_DIR/testresults
+sudo chmod +x create-summary-csv.sh
+sudo ./create-summary-csv.sh -n Passthrough HTTP service -x
 
```

Comment:
Let's call the MD file creation task here, instead of being inside `create-summary-csv`.